components/Benefit/Benefit.tsx

```diff
@@ -0,0 +1,17 @@
+import type { NextPage } from "next";
+import styles from "styles/Benefit.module.scss";
+
+type BenefitProps = {
+    text: string;
+};
+
+const Benefit: NextPage<BenefitProps> = ({ text }) => {
+    return (
+        <li className={styles.benefit}>
```

Comment:
suggestion: personally I’d not have made a separate component for this, since it feels like this won’t ever be used outside of BenefitsList.